gallery_dl/extractor/weasyl.py

```diff
@@ -78,37 +93,34 @@ class WeasylSubmissionExtractor(WeasylExtractor):
 
     def __init__(self, match):
         WeasylExtractor.__init__(self, match)
-        self.submitid = int(match.group(1))
+        self.submitid = int(match[1])
         if len(match.groups()) == 3:
-            self.title = match.group(2)
+            self.title = match[2]
```

Comment:
Using angle brackets here is only supported in Python3.6+.
Keep the `match.group()` syntax for Python3.4 and 3.5 compatibility.